app/assets/security 2/10k-common-passwords

```diff
@@ -0,0 +1,10000 @@
+password
```

Comment:
This file shouldn't be in this PR, should it?

Reply:
I guess because of merge, this might have come. Will look into this.